pkg/cmd/bootstrap.go

```diff
@@ -171,23 +179,28 @@ func initiateInteractiveMode(io *BootstrapParameters, client *utility.Client) er
 	}
 	io.GitOpsWebhookSecret = ui.EnterGitWebhookSecret()
```

Comment:
Make sure the two questions have different messages (one for git repo and the other one is service repo)